### ISSUE TITLE
Fix Focal Loss input/output for upcoming version (temporal solution in master)

### ISSUE THREAD

**Issue by jrzaurin** [jrzaurin/pytorch-widedeep#14]
The issue is related to the use of `F.binary_cross_entropy_with_logits` instead of `F.binary_cross_entropy`. The problem is that I use a sigmoid activation in the case of a binary classification and no activation for a multiclass problem. Therefore, although the use of `F.binary_cross_entropy_with_logits` is correct for multiclass, is not the case for binary.

also the method `get_weight` applies a `sigmoid` activation to the entries, which is only correct for multiclass. 

Need to fix all this

See here:
https://github.com/jrzaurin/pytorch-widedeep/blob/7c388b01bcf4a3ba1eb9e93fe187d938fc9b4432/pytorch_widedeep/models/wide_deep.py#L746

**Comment by jrzaurin** [comment]
This is fixed in the new release 0.4.1
